synth.metadata

```diff
@@ -3,280 +3,27 @@
     {
```

Comment:
Can you remove the modifications to this file from your branch? We don't want to lose the `generatedFiles` information.

Reply:
Done.